crates/re_viewport/src/viewport_blueprint_ui.rs

```diff
@@ -388,16 +405,23 @@ fn focus_tab(tree: &mut egui_tiles::Tree<SpaceViewId>, tab: &SpaceViewId) {
 /// and show a visibility button if the row is hovered.
 ///
 /// Returns true if visibility changed.
+#[allow(clippy::fn_params_excessive_bools)]
 fn blueprint_row_with_buttons(
     re_ui: &re_ui::ReUi,
     ui: &mut egui::Ui,
     enabled: bool,
     visible: bool,
+    selected: bool,
     add_content: impl FnOnce(&mut egui::Ui) -> egui::Response,
     add_on_hover_buttons: impl FnOnce(&re_ui::ReUi, &mut egui::Ui),
 ) {
     let where_to_add_hover_rect = ui.painter().add(egui::Shape::Noop);
 
+    ui.visuals_mut().widgets.hovered.expansion = 2.0;
+    ui.visuals_mut().widgets.active.expansion = 2.0;
+    ui.visuals_mut().widgets.open.expansion = 2.0;
+    ui.visuals_mut().widgets.inactive.expansion = 2.0;
```

Comment:
these visuals changes now leak into the caller which is generally a bad thing, they should be scoped to this method

In this case it should be enough to just move them insode the `with_layout` block (?). Otherwise, `ui.scope` is the way to go

Reply:
Removed altogether as per our discussion.